src/main/java/mate/academy/onlinebookstore01/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.onlinebookstore01.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.onlinebookstore01.validation.FieldMatch;
+
+@Data
+public class UserRegistrationRequestDto {
+    private static final String EMAIL_EXCEPTION
+            = "Cannot be null/empty and must consist symbol: @";
+    private static final String PASSWORD_EXCEPTION
+            = "Cannot be null and must be between 8 and 16 characters.";
+    private static final String NULL_MESSAGE
+            = "Cannot be null or empty";
+    @NotNull
+    @NotEmpty

```

Comment:
It's no need to use `@NonNull` with `@NotEmpty`.
https://www.baeldung.com/java-bean-validation-not-null-empty-blank#a-side-by-side-comparison
Also, think about using `@NotBlank`.